packages/react-bindings/src/compose.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+
+type ComposeOptions = {
+  // TODO: better typings PLZ
+  className?: string
+  displayName: string
+  mapPropsToBehavior?: Function
+  mapPropsToStyles?: Function
+  handledProps?: string[]
+  overrideStyles?: boolean
+}
+
+const COMPOSE_CONFIG_PROP_NAME = '__unstable_config'
```

Comment:
Should this go in the `react-theming` package? I thought that was where we were going to have compositional utilities.

We need a package which partners can easily use to compose components and provide a theme, with the least amount of other dependencies in it. This will allow a solid foundation to exist for building out an ecosystem of components.

Reply:
I don't think that composition covers only styling aspects 🤔